python/packages/devui/agent_framework_devui/_mapper.py

```diff
@@ -280,59 +278,6 @@ async def _convert_workflow_event(self, event: Any, context: dict[str, Any]) ->
             logger.warning(f"Error converting workflow event: {e}")
             return [await self._create_error_event(str(e), context)]
 
-    def _serialize_payload(self, value: Any) -> Any:
```

Comment:
I just added this yesterday because there were serialization errors while running Agent Executors.